### ISSUE TITLE
안녕하세요 지나가던 대학생입니다.

### ISSUE THREAD

**Issue by hwan0813** [keon/3-min-pytorch#1]
지나가다가 우연히 봤어요. 오류가 있는거같아서, 혹시나 제 생각이 맞으면 수정하시면
좋을것 같습니다. 

4번 챕터에 03-overfitting-and-regularization.py 및 주피터 파일에서

def train(model, train_loader, optimizer):
    model.train()
    for data, target in enumerate(train_loader):
        data, target = data.to(DEVICE), target.to(DEVICE)
        optimizer.zero_grad()
        output = model(data)
        loss = F.cross_entropy(output, target)
        loss.backward()
        optimizer.step()

3번째 줄 에러인것 같습니다. enumerate 에서 인덱스가 data로 들어가는거 같네요
for batch_index, (data, target) in enumerate(train_loader): 이 맞는것 같습니다

**Comment by keon** [comment]
안녕하세요 ~
아직 홍보한적이 없는데 어떻게 찾으셨나요? ㅎㅎ 관심 감사합니다.
이 레포 전체적으로 전부 준비가 덜돼서 튜닝도 안돼있고 여기저기 고칠곳이 많습니다.
제보하신 오류는 맞습니다. 수정하도록 하겠습니다 :)

**Comment by hwan0813** [comment]
제가 파이토치 이제 막 입문해서 깃 찾아보다가 발견했습니다.
안그래도 3분시리즈 진짜 좋아하는데 3분 파이토치라니.. 진짜 나오면 바로 살거에요 ㅋㅋㅋㅋ
 
-----Original Message-----

**Comment by hellojinwoo** [comment]
저도 사고 싶은데 아직 안나왔네요 ㅠㅠ
